modules/learn/pages/services-and-indexes/indexes/storage-modes.adoc

```diff
@@ -2,24 +2,27 @@
 :page-aliases: indexes:storage-modes,understanding-couchbase:services-and-indexes/indexes/storage-modes,architecture:index-storage
 
 [abstract]
-A Secondary Index can be saved in either of two ways: _standard_ or _memory-optimized_.
+A Secondary Index can be saved in either of two ways: _memory-optimized_ or _standard_.
 
 [#memopt-gsi2]
 == Memory-Optimized Index Storage
 
-[NOTE]
-.https://www.couchbase.com/products/editions[ENTERPRISE EDITION]
-====
+[.labels]
+[.edition]#{enterprise}#
+
 Memory-optimized index-storage is supported by the _Nitro_ storage engine, and is only available in Couchbase Server Enterprise Edition.
-====
 
 Memory-optimized index-storage allows high-speed maintenance and scanning; since the index is kept fully in memory at all times.
 A snapshot of the index is maintained on disk, to permit rapid recovery if node-failures are experienced.
 To be consistently beneficial, memory-optimized index-storage requires that all nodes running the Index Service have a memory quota sufficient for the number and size of their resident indexes, and for the frequency with which the indexes will be updated and scanned.
 
 Memory-optimized index-storage may be less suitable for nodes where memory is constrained; since whenever the Index Service memory-quota is exceeded, indexes on the node can neither be updated nor scanned.
-In such circumstances, an error-notification is provided; and although the indexes remain in `ONLINE` state, traffic is routed away from the node.
+
+If indexer RAM usage goes above 75%, an xref:manage:manage-settings/configure-alerts.adoc[error-notification] is provided.
```

Comment:
It's nitpicky, but could we sat "of the quota" either for each of the percentages, or maybe a more overarching header along the lines of "The indexer will take action if certain percentage thresholds of its quota are reached:"?

Otherwise, it's a tad open to interpretation - is this 75% of the system RAM for example...

Reply:
Updated